app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less

```diff
@@ -727,6 +727,13 @@
                     position: static;
                 }
             }
+            .content {
+                .fieldset {
+                    .actions-toolbar {
```

Comment:
`.actions-toolbar` selector receives `width: 1%` from `Magento_Paypal/web/css/source/module/_review.less`. Can you check it more thoroughly?

Please also check if this works with just `.block { .actions-toolbar` selector path (remove .content and .fieldset selectors). Add empty lines before and after styles.

Reply:
@ptylek, thank you for your revision changes that have been done and pushed.